### DISCUSSION TITLE
Getting usage billing emails but dashboard says I’m okay

### DISCUSSION THREAD

**Discussion by preetpatel** [vercel/community#2662] [Help] [answered]
### Summary

Hi,
I’ve got a Vercel KV set up on one of my projects. A little while ago I got two emails: one for exceeding the kv 512mb data transfer and the other for exceeding 150,000 requests 

I went and looked on my dashboard and on there I can only see 21k requests and 200mb data transfer.
Which should I trust?

### Steps to Reproduce

Visit dashboard for vercel

**Comment by amyegan** [comment]
Sorry for the confusion this has caused, @preetpatel. Is this for a Pro team or for a Hobby account?

**Reply by preetpatel** [reply]
Pro team @amyegan

**Reply by amyegan** [reply]
If you share the team slug or id, I can look up the usage and make sure you aren't being billed incorrectly

**Reply by preetpatel** [reply]
Thanks @amyegan . Our team ID is `team_iGuEh4fs5TwGYiqdudzurZro`

**Reply by amyegan** [reply, marked as answer]
It looks like there was a problem with the billing notifications, and your dashboard is showing the correct numbers. If you still have an open invoice or have already been charged, please [submit a Billing case](https://vercel.com/help#issues) to the Customer Success Team. They'll be able to discuss account and payment details with you safely outside of this public discussion to set things right.

**Comment by jkmartindale** [comment]
Just ran into the same issue...glad to know I'm not alone, I guess?

**Reply by amyegan** [reply]
The original issue was resolved. If you have more recently received a billing notification that does not match your dashboard, please [submit a case](https://vercel.com/help#issues) to the Customer Success Team so we can investigate 🙏

**Reply by JOsacky** [reply]
Hello, Seems like a similar issue popped up with bandwidth usage. The dashboard stopped counting bandwidth around July 13th but I got a billing email saying I'm being charged for bandwidth. I submitted a case with customer success team and wanted to flag here too in case others are experiencing this issue again

**Reply by JOsacky** [reply]
The dashboard was fixed yesterday and backfilled to all the days it was broken. Thanks for fixing!
